WordPress/openverse
# Update appearance of sensitive image thumbnails

## Fixes



Fixes #4777 by @fcoveram 

## Description



This PR
- adds a new background token "blur" that corresponds to white-88 or black-72 depending on the theme
  - removes an incorrect, and redundant, use of `bg-blur`
- updates the image cell to show the text "Sensitive content" and an icon indicating that the image has been blurred because of sensitivity

## Testing Instructions


0. Check out the PR and run the Nuxt server.
1. Go to `/preferences` and check the "fake_sensitive" preference.
2. Perform a search for something.
3. Verify that a number of results have been blurred and show the "hide" icon.
4. Hover or focus over these results to reveal the bottom-centered "Sensitive content" note instead of the license info.
5. Switch to a mobile view.
6. Verify that the results have the "hide" icon and show the "Sensitive content" note in place of the license info.

## Checklist


      PRs) or the media properties generator (`ov just catalog/generate-docs media-props`
      for the catalog or `ov just api/generate-docs` for the API) where applicable.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
